Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/NolPay/NolPayLinkedCardsComponent.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  NolPayLinkedCardsComponent.swift
+//  PrimerSDK
+//
+//  Created by Boris on 15.9.23..
+//
+
+import Foundation
+#if canImport(PrimerNolPaySDK)
+import PrimerNolPaySDK
+#endif
+
+public class NolPayLinkedCardsComponent: PrimerHeadlessComponent {
+    
+    var nolPay: PrimerNolPayProtocol?
```

Comment:
This fails for developers who do not include the Nol pay dependency. It needs to be behind `#if canImport(PrimerNolPaySDK)`